### PR TITLE
Add option to rename origin of output records

### PR DESCRIPTION
**Pull request by ypc-faros**

### CODE REVIEW

File: sources/faros-graphql-source/resources/spec.json [hunk 1]
```diff
@@ -69,6 +69,17 @@
           "Nested",
           "Flat"
         ]
+      },
+      "map_origin": {
+        "order": 7,
+        "type": "string",
+        "title": "Map origin",
```
Comment (tovbinm) [inline sources/faros-graphql-source/resources/spec.json:76]: `Map origin` -> `Replace Records Origin`

File: sources/faros-graphql-source/resources/spec.json [hunk 2]
```diff
@@ -69,6 +69,17 @@
...
+        "description": "JSON map where each entry (origin1, origin2) indicates that the output records' origin should be replaced from 'origin1' by 'origin2'",
```
Comment (tovbinm) [inline sources/faros-graphql-source/resources/spec.json:77]: `JSON map to replace records origin values, where each entry (origin1, origin2) indicates that the output records' origin property should be replaced from 'origin1' by 'origin2'`

File: sources/faros-graphql-source/resources/spec.json [hunk 3]
```diff
@@ -69,6 +69,17 @@
           "Nested",
           "Flat"
         ]
+      },
+      "map_origin": {
```
Comment (tovbinm) [inline sources/faros-graphql-source/resources/spec.json:73]: `map_origin` -> `replace_origin_map`

File: sources/faros-graphql-source/src/streams/faros-graph.ts [hunk 4]
```diff
@@ -240,6 +247,10 @@ export class FarosGraph extends AirbyteStreamBase {
       // We only use these fields for updating the incremental state
       const result = omit(item, ['metadata', 'refreshedAt']);
 
+      if (_.has(result, 'origin')) {
```
Comment (tovbinm) [inline sources/faros-graphql-source/src/streams/faros-graph.ts:250]: `if (this.originRemapper && _.has(result, 'origin'))`

**Review by tovbinm — APPROVED** [review]

**Comment by ypc-faros** [comment]
Thank you @tovbinm 🙏 !

**Pull request merged**